_data/store/simonschaufi-customer-email-report-bundle.yml

```diff
@@ -0,0 +1,13 @@
+developer: simonschaufi
+icon: fas fa-home
```

Comment:
```suggestion
icon: fas fa-envelope
```

Reply:
Where is this icon used? And why did you remove the editorconfig?